R/hooks.R

```diff
@@ -142,19 +143,22 @@ log_errors <- function(muffle = getOption('logger_muffle_errors', FALSE)) {
 log_shiny_input_changes <- function(input,
                                     level = INFO,
                                     namespace = NA_character_,
-                                    excluded_inputs = character()) {
+                                    excluded_inputs = character(),
+                                    session = NULL) {
+
+    ns <- if (!is.null(session)) session$ns(character(0))
```

Comment:
I think it would make more sense to put the usage of `session` object after we check that it's correct (currently: line 153).

Reply:
101f86984d2ab8849e7a959142948713de4638dd